newsletter/_posts/2019-04-01-Jlab-Workshop.md

```diff
@@ -0,0 +1,122 @@
+---
+title: HSF-OSG-WLCG Workshop at Jefferson Lab, HOW2019
+author: Graeme Stewart
+layout: newsletter
+---
+
+The annual [HEP Software Foundation
+workshop](https://indico.cern.ch/event/759388/){:target="jlab_agenda"} happened
+this year in [Jefferson
+Laboratory](https://www.jlab.org){:target="jlab_webpage"}  from the 18th to 22nd
+of March. This year we had the opportunity to join forces again with the
+Worldwide LHC Computing Grid (WLCG) community and the  US Open Science Grid
+(OSG). Almost 250 scientists, from LHC, HEP and non-HEP communities joined the
+meeting.
+
+On the first day we opened the meeting with an excellent introduction to JLab
+from lab director Stuart Henderson and an overview of JLab computing and
+software from Raffaella De Vita. That was followed by plenary talks from the LHC
+experiments, other HEP experiments and many non HEP communities, including the
+next generation US nuclear physics experiment, the Electron-Ion Collider. That
+set the stage for the computing and software challenges we face in data
+intensive science for the next decade.
+
+<div style="text-align:center; padding:25px; float:left">
+<img src ="/images/workshops/jlab-group.jpg" alt="JLab Workshop Group Photo" />
+<br>Workshop Participants. Photo &copy; DOE Jefferson Laboratory
+</div>
+
+The theme of working more closely with other sciences was underlined by the
+discussion on the *Evolution of the WLCG Collaboration* on Monday afternoon.
+Sharing an infrastructure for big data sciences, building on what we know and
+already have, received wide support, but the details of how to manage this, for
+all communities, need to be worked out.
+
+<div style="text-align:center; padding:25px; float:right">
+<img src ="/images/workshops/jlab-plenary1.jpg" alt="JLab Workshop Group Photo" />
+<br>Plenary Discussions. Photo &copy; DOE Jefferson Laboratory
+</div>
+
+Common sessions for HSF and WLCG on Tuesday looked at the evolution of
+technology - processors, storage and networking - and how HEP is managing to
```

Comment:
After 'networking -', I'd add 'based on the impressive work done by the [HEPiX Benchmarking WG](http://w3.hepix.org/benchmarking.html)'